src/main/java/edu/ucla/library/libcal/handlers/ProxyHandler.java

```diff
@@ -73,12 +73,12 @@ public void handle(final RoutingContext aContext) {
         final String path = aContext.request().path();
 
         try {
-            if (path.endsWith(LIBCAL) || path.endsWith(LIBCAL.concat(SLASH))) {
+            if (path.equals(EMPTY) || path.equals(SLASH)) {
```

Comment:
It isn't possible for the path to equal the empty string; even if you call a WebClient on the "path" `""`, it converts that to `"/"`.

Even if we just checked for the root path, though, this if-block is getting us into request validation territory, which I don't think our application should be concerned with.